lib/ex_pesa/Jenga/account_services/account_balance.ex

```diff
@@ -0,0 +1,61 @@
+defmodule ExPesa.Jenga.AccountServices.AccountBalance do
+  @moduledoc """
+  This enables your application to retrieve the current and available balance of an account.
+  """
+
+  import ExPesa.Jenga.JengaBase
+  alias ExPesa.Jenga.Signature
+
+  @uri_affix "/accounts/v2/accounts/balances/"
+  @current_valid_countries ~w(KE KE TZ UG RW DRC SS)
```

Comment:
@r-coh  we may miss updates incase this changes. is this the best approach or should we leave it open?

Reply:
It can be left open